diff --git a/src/catppuccin-macchiato/Components/PasswordField.qml b/src/catppuccin-macchiato/Components/PasswordField.qml
index 0460255..640e43f 100644
--- a/src/catppuccin-macchiato/Components/PasswordField.qml
+++ b/src/catppuccin-macchiato/Components/PasswordField.qml
@@ -8,7 +8,6 @@ TextField {
   placeholderText: "Password"
   echoMode: TextInput.Password
   passwordCharacter: "•"
-  passwordMaskDelay: 1000
   selectionColor: config.overlay0
   renderType: Text.NativeRendering
   font.family: config.Font
